'''    The try block lets you test a block of code for errors.

    The except block lets you handle the error.

    The else block lets you execute code when there is no error.

    The finally block lets you execute code, regardless of the result of the try- and except blocks.    '''

# try:
#     Name = input("What's your name?: ")
#     if Name.isalpha() == False or not Name.isalpha():
#         raise ValueError 
#     else:
#         print ("Hey", Name)

# except ValueError or TypeError:
#     print("Error: Name must be a string.")



f = open("randompythonfile.txt")


